### ISSUE TITLE
KL loss implementation is less effective

### ISSUE THREAD

**Issue by YuriCat** [google-research/seed_rl#81]
We tried your implementation of KL regularization loss, but it didn't worked well.
That is because this KL loss further increases the probability of an action (chosen by chance in episode generation).

I think there are better ways:
```
t: target_action_log_probs
b: behaviour_action_log_probs
```
### Forward KL
```
kl_loss = stop_gradient(exp(t - b) - 1) * t
```
### Reverse KL
```
kl_loss = stop_gradient((b - t) * exp(t - b)) * -t
```

While value clipping may be necessary in practice, these losses are unbiased in principle.

This is my experimental code:
https://github.com/YuriCat/test/blob/master/algo/kl_loss.py
